release_notes/ocp_3_6_release_notes.adoc

```diff
@@ -0,0 +1,2419 @@
+
+[NOTE]
+====
+HAProxy can only terminate IPv6 traffic when the router uses the network stack
+of the host (default). When using the container network stack (`oadm router
+--service-account=router --host-network=false`), there is no global IPv6 address
+for the pod.
+====
+
+[[ocp-36-installation]]
+=== Installation
+
+[[ocp-36-automated-installtion-of-cloudforms]]
+==== Automated installation of CloudForms 4.5 Inside OpenShift (Technology Preview)
+
+The installation of containerized CloudForms inside {product-title} is now part
+of the main installer (currently in xref:ocp-36-technology-preview[Technology
+Preview]). It is now treated like other common components (metrics, logging, and
+so on).
+
+After the {product-title} cluster is provisioned, there is an additional
+playbook you can run to deploy CloudForms into the environment (using the
+`openshift_cfme_install_app flag` in the hosts file).
+
+----
+$ ansible-playbook -v -i <INVENTORY_FILE> playbooks/byo/openshift-cfme/config.yml
+----
+
+Requirements:
+
+[cols="4*", options="header"]
+|===
+|Type
+|Size
+|CPUs
+|Memory
+
+|Masters
+|1+
+|8
+|12 GB
+
+|Nodes
+|2+
+|4
+|8 GB
+
+|PV Storage
+|25 GB
+|N/A
+|N/A
+|===
+
+[NOTE]
+====
+NFS is the only storage option for the Postgres database at this time.
+====
+
+[[ocp-36-installation-of-etcd-docker-daemon-and-ansible-installer-as-system-containers]]
+==== Installation of etcd, Docker Daemon, and Ansible Installer as System Containers (Technology Preview)
+
+RHEL System Containers offer more control over the life cycle of the services
+that do not run inside {product-title} or Kubernetes. Additional system
+containers will be offered over time.
+
+System Containers leverage the OSTree on RHEL or Atomic Host. They are
+controlled by the kernel init system and therefore can be leveraged earlier in
+the boot sequence. This feature is enabled in the installer configuration.
+
+----
+$ atomic install --system --set INVENTORY_FILE=$(pwd)/inventory registry:port/openshift3/ose-ansible:v3.6
+
+$ systemctl start ose-ansible-v3.6
+----
+
+[IMPORTANT]
+====
+This feature (currently in xref:ocp-36-technology-preview[Technology Preview)] is
+currently not for production workloads and must be used in a non-production
+sandbox.
+====
+
+For more information, see
+xref:../install_config/install/advanced_install.adoc#advanced-install-configuring-system-containers[Configuring
+System Containers].
+
+[[ocp-36-etcd3-model-for-new-installations]]
+==== etcd3 Data Model for New Installations
+
+Starting with new installations of {product-title} 3.6, the etcd3 v3 data model
+is the default. Also, a migration script is released to allow those who upgraded
+to {product-title} 3.6 to migrate to the etcd3 v3 data model.
+
+Then, in {product-title} 3.7,  the upgrade will stop if the installer detects
+data is not migrated to the etcd3 data model. This will be a mandatory and
+manual step.
+
+This is controlled by the `openshift_master_disable_etcdv3=true` setting.
```

Comment:
Yeah, see my comment above. Given we've decided not to advise customers to migrate to v3 in 3.6.0 GA we need to rewrite this entire section.